processor/pom.xml

```diff
@@ -57,7 +57,12 @@
             <artifactId>mapstruct</artifactId>
             <scope>provided</scope>
         </dependency>
-
+        <dependency>
+            <groupId>org.eclipse.tycho</groupId>
```

Comment:
Ok, that's cool. It's not quite clear to me why it works, but maybe annotation processor JARs are treated "special" by Eclipse's OSGi environment in some way.